src/Component/Nav/NavScroll.js

```diff
@@ -0,0 +1,201 @@
+import React, { useState, useEffect } from 'react'
+import styled, { StyleSheetManager } from "styled-components";
+import Dropdown from './Dropdown';
+import { AiOutlineDownload } from 'react-icons/ai';
+import { FaUser } from 'react-icons/fa';
+import { FaHeart } from 'react-icons/fa';
+import { AiFillShopping } from 'react-icons/ai';
+import { RiLoginBoxFill } from 'react-icons/ri';
+import { FaSearch } from 'react-icons/fa';
+
+export default function NavScroll(props) {
+  const [navData, setNavData] = useState([]);
+  const [isHover, setIsHover] = useState(false);
+
+  const navMenuData = navData[0]?.data;
+  const navDetailMenuL = navData[1]?.data;
+  const navDetailMenuR = navData[2]?.data;
```

Comment:
```suggestion
const [navMenu, navDetailMenuL, navDetailMenuR] = navData;
```
이렇게 비구조화 하실 수 있습니다.
그리고 아래에서는 
```
navMenu?.data, 
navDetailMenuL?.data,
navDetailMenuR?.data
```

이렇게 쓰실 수 있겠네요!